content/automate/ManageIQ/Infrastructure/VM/Provisioning/Placement.class/__methods__/microsoft_best_fit_least_utilized.rb

```diff
@@ -31,7 +35,7 @@
 end
 
 # Set host and storage
-prov.set_host(host) if host
+host ? prov.set_host(host) : prov.set_option(:placement_host_name, [nil, nil])
```

Comment:
Minor, but I suggest you move the `prov.set_option(:placement_host_name, [nil, nil])` logic into a method named `clear_host` which would make this easier to read.

```ruby
host ? prov.set_host(host) : clear_host(prov)
...

def clear_host(prov)
  prov.set_option(:placement_host_name, [nil, nil])
end
```